src/amo/pages/Addon/index.js

```diff
@@ -477,7 +477,12 @@ export class AddonBase extends React.Component {
                 {i18n.gettext('Extension Metadata')}
               </h2>
             </header>
-            {addon && <InstallWarning addon={addon} />}
+            {addon && (
+              <InstallWarning
+                addon={addon}
+                className="Addon-install-warning-notice"
```

Comment:
I believe that we can use `width: 100%` for `InstallWarning` everywhere, so we don't really need to allow for a `className`for `InstallWarning`. We can just change its default style.